test/helpers/compile_deploy.js

```diff
@@ -0,0 +1,50 @@
+const { compile } = require("solc");
```

Comment:
Perhaps we can export `compile`, `deploy` and `compileDeploy`. I try to avoid using dashes as it is error prone when referenced within javascript. I'm in favor of camelCase.

Reply:
File names with dashes are fine in JS, since they will always be in a string. Additionally, in the future we'll likely be breaking ganache-core into smaller modules that are then published to npm; multi-word packages will most likely be published with dashes, so I think matching file names to published names makes sense there.

I don't like "compileDeploy" or "compile_deploy" because it is missing a word.

Reply:
I can change this to `compileAndDeploy`.